sig-creation-procedure.md

```diff
@@ -0,0 +1,83 @@
+### SIG creation procedure
+
+#### 
+**Create the GitHub teams**
```

Comment:
I would put the team creation last, so that you can mention that the users should be added to the teams.